database/migrations/2018_11_06_213626_create_fields_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFieldsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('fields', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
```

Comment:
we probably need english name and dutch name

Reply:
ok